prebid/native-implementation.md

```diff
@@ -101,15 +102,16 @@ Specific bidders may not support all of the fields listed below or may return di
 
 {% include dev-docs/native-image-asset-sizes.md %}
 
-### 3.2 Native custom assets
+### 3.2 Custom native assets
 
-Prebid.js allows custom assets in order to fit some bidder's requirements through the special `mediaTypes.native.ext` key. The assets under this key will be whitelisted and could be retrieved during the rendering.
+In order to fit special bidder requirements, Prebid.js supports assets beyond the standard set. Define custom attributes at mediaTypes.native.ext. When `sendTargetingKeys` is false, the assets under the `native.ext` key can be retrieved during the render.
```

Comment:
> When `sendTargetingKeys` is false, 

There is nothing in the code to avoid a `ext` key to not be retrieved if `sendTargetingKeys` is true.

Reply:
ok, removed that note.